index.js

```diff
@@ -11,11 +11,15 @@ const {
   toStateless
 } = require('@creuna/react-scripts');
 
-const pkg = require('./package.json');
+const configstore = require('./source/configstore');
```

Comment:
Can we abstract this? We really only need the package version and not the configstore api here